openedx/core/djangoapps/site_configuration/models.py

```diff
@@ -60,6 +60,25 @@ def get_value(self, name, default=None):
 
         return default
 
+    @classmethod
+    def get_configuration_for_org(cls, org):
```

Comment:
this makes sense, but we should probably have a test for it?

Reply:
Yeah, we should. I'll add one.